mathjax3-ts/output/chtml/FontData.ts

```diff
@@ -16,28 +16,377 @@
+        const data = this.getChar('normal', n) || this.getChar('-size4', n);
+        const css: StyleData = {content: '"' + this.char(n, true) + '"'};
+        if (part !== 'ext') {
+            css.padding = this.em0(data[0]) + ' 0 ' + this.em0(data[1]);
+        }
+        styles[this.cssRoot + 'mjx-stretchy-v[c="' + c + '"] mjx-' + part + ' mjx-c::before'] = css;
+        return data[0] + data[1];
+    }
+
+    /*******************************************************/
+
+    /**
+     * @param {StyleList} styles         The style object to add styles to
+     * @param {string} c                 The delimiter character string
+     * @param {CHTMLDelimiterData} data  The data for the delimiter whose CSS is to be added
+     */
+    protected addDelimiterHStyles(styles: StyleList, c: string, data: CHTMLDelimiterData) {
+        const [beg, ext, end, mid] = data.stretch;
+        this.addDelimiterHPart(styles, c, 'beg', beg);
+        this.addDelimiterHPart(styles, c, 'ext', ext, !(beg || end));
+        this.addDelimiterHPart(styles, c, 'end', end);
+        if (mid) {
+            this.addDelimiterHPart(styles, c, 'mid', mid);
+            styles[this.cssRoot + 'mjx-stretchy-h[c="' + c + '"] > mjx-ext'] = {width: '50%'};
+        }
+    }
+
+    /**
+     * @param {StyleList} styles  The style object to add styles to
+     * @param {string} c          The vertical character whose part is being added
+     * @param {string} part       The name of the part (beg, ext, end, mid) that is being added
+     * @param {number} n          The unicode character to use for the part
+     */
+    protected addDelimiterHPart(styles: StyleList, c: string, part: string, n: number, force: boolean = false) {
+        if (!n) {
+            return 0;
+        }
+        const data = this.getChar('normal', n) || this.getChar('-size4', n);
+        const options = data[3] as CHTMLCharOptions;
+        const C = (options && options.c ? options.c : this.char(n, true));
+        const css: StyleData = {content: '"' + C + '"'};
+        if (part !== 'ext' || force) {
+          css.padding = this.em0(data[0]) + ' 0 ' + this.em0(data[1]);
+        }
+        styles[this.cssRoot + 'mjx-stretchy-h[c="' + c + '"] mjx-' + part + ' mjx-c::before'] = css;
+    }
+
+    /*******************************************************/
+
+    /**
+     * @param {StyleList} styles  The style object to add styles to
+     * @param {string} vclass     The variant class string (e.g., .mjx-b) where this character is being defined
+     * @param {number} n          The unicode character being defined
+     * @param {CharData} data     The bounding box data and options for the character
+     */
+    protected addCharStyles(styles: StyleList, vclass: string, n: number, data: CHTMLCharData) {
+        const [h, d, w, options] = data as [number, number, number, CHTMLCharOptions];
+        const css: StyleData = {};
+        if (options.css) {
+            if (options.css & CSS.width) {
```

Comment:
At this point `options.css` exists.

Reply:
Note that this is `&` not `&&`.  The `options.css` property is a bit field, and we are testing for the presence of a specific bit in that field, both here and the ones marked below.  The `CSS` object is a collection of named bit positions.